examples/solids/qfunctions/FSCurrent-NH1.h

```diff
@@ -85,66 +85,79 @@ CEED_QFUNCTION_HELPER CeedScalar computeJM1(const CeedScalar grad_u[3][3]) {
 };
 #endif
 
+// -----------------------------------------------------------------------------
+// Compute matrix^(-1), where matrix is nonsymetric, returns array of 9
+// -----------------------------------------------------------------------------
+#ifndef MatinvNonSym
+#define MatinvNonSym
+CEED_QFUNCTION_HELPER int computeMatinvNonSym(const CeedScalar A[3][3],
+    const CeedScalar detA, CeedScalar Ainv[9]) {
+  // Compute A^(-1) : A-Inverse
+  CeedScalar B[9] = {A[1][1]*A[2][2] - A[1][2]*A[2][1], /* *NOPAD* */
+                     A[0][0]*A[2][2] - A[0][2]*A[2][0], /* *NOPAD* */
+                     A[0][0]*A[1][1] - A[0][1]*A[1][0], /* *NOPAD* */
+                     A[0][2]*A[1][0] - A[0][0]*A[1][2], /* *NOPAD* */
+                     A[0][1]*A[1][2] - A[0][2]*A[1][1], /* *NOPAD* */
+                     A[0][2]*A[2][1] - A[0][1]*A[2][2], /* *NOPAD* */
+                     A[0][1]*A[2][0] - A[0][0]*A[2][1], /* *NOPAD* */
+                     A[1][0]*A[2][1] - A[1][1]*A[2][0], /* *NOPAD* */
+                     A[1][2]*A[2][0] - A[1][0]*A[2][2]  /* *NOPAD* */
+                    };
+  for (CeedInt m = 0; m < 9; m++)
+    Ainv[m] = B[m] / (detA);
+
+  return 0;
+};
+#endif
+
 // -----------------------------------------------------------------------------
 // Common computations between Ftau and dFtau
 // -----------------------------------------------------------------------------
 CEED_QFUNCTION_HELPER int commonFtau(const CeedScalar lambda,
-                                     const CeedScalar mu, const CeedScalar grad_u[3][3],
```

Comment:
The underscore for `F_inv` was intentionally added. I'm not sure why `Grad_` should be capitalized here - it isn't in other QFunctions

Reply:
I haven't thoroughly made every variable in all of the different solids QFunctions match our style: https://github.com/CEED/libCEED/issues/729

Reply:
That makes sense if it is typical. I'm not an expert in continuum mechanics notation so I'm not sure what the common convention would be in that case.

We still are missing a bunch of underscores between words though. `Finv` vs `F_inv` and `tempGradu` vs `temp_Grad_u` though

Reply:
I wouldn't stress about this because I think we'll be refactoring this anyway to decouple material from structural concerns. (The exact way in which we refactor depends on empirical tests of what optimizations we can rely on compilers to do.)